### PR TITLE
Refactor of Habitat Slackin Landing Page

### PR DESCRIPTION
**Pull request by eeyun**
First iteration on exposing community resources as part of the Slack signup page!

Currently we are still using thom's animations, but all content and links have been updated to point to habitat related resources. On a longer timeline we should replace these with animations specific to Habitat and more useful community links.

Thank you @thommay for all of your work here.

### CODE REVIEW

**Comment by tas50** [comment]
:shipit:

File: lib/assets/index.md [hunk 1]
```diff
@@ -95,24 +94,20 @@ less critical than they originally thought.
...
-  * You can find a permanent, searchable archive of our Slack channels at [SlackArchive](http://chefcommunity.slackarchive.io/).
+  * You can find a permanent, searchable archive of our Slack channels at [SlackArchive](http://habitat-sh.slackarchive.io/general/).
 You can also search the last ten thousand messages from within the Slack client. Be aware that archiving must be configured
 for each channel by inviting the `@slack_archive_bot` account.
-* **Where Else Can I Find Help With Chef?**
-  * We have a mailing list and forum at [discourse.chef.io](https://discourse.chef.io/)
-if you prefer a more asynchronous approach. We also have volunteers on
-[StackOverflow](https://stackoverflow.com/tags/chef). If volunteer support isn't
+* **Where Else Can I Find Help With Habitat?**
+  * We have a mailing list and forum at [forums.habitat.sh](https://forums.habitat.sh/)
```
Comment (juliandunn) [inline lib/assets/index.md]: We don't need to say in Habitat-land that it's a mailing list -- that was for semantic "backwards compat" on the Chef Discourse. We really only have a forum :)

Comment (eeyun) [reply lib/assets/index.md]: Awesome, this has been updated.

**Comment by eeyun** [comment]
I dont think @thesentinels is watching this repo currently. But I can try:

@thesentinels approve

EDIT: Nope, gonna merge since 4x thumbs.

**Pull request merged**